src/org/intellij/lang/batch/runner/BatchRunner.java

```diff
@@ -35,7 +35,7 @@ protected RunContentDescriptor doExecute(final Project project,
         ExecutionResult executionResult = state.execute(env.getExecutor(), this);
         if (executionResult == null || executionResult.getExecutionConsole() == null) return null;
 
-        final RunContentBuilder contentBuilder = new RunContentBuilder(this, executionResult, env);
+        final RunContentBuilder contentBuilder = new RunContentBuilder(/*this,*/ executionResult, env);

```

Comment:
Yes, feel free to delete in a subsequent commit after you accept the PR. I just forgot about it.